bundles/org.openhab.binding.rfxcom/src/test/java/org/openhab/binding/rfxcom/internal/messages/RFXComUndecodedRFMessageTest.java

```diff
@@ -51,12 +50,21 @@ public void testSomeMessages() throws RFXComException {
 
     @Test
     public void testLongMessage() throws RFXComException {
-        RFXComUndecodedRFMessage msg = (RFXComUndecodedRFMessage) RFXComMessageFactoryImpl.INSTANCE
-                .createMessage(PacketType.UNDECODED_RF_MESSAGE);
-        msg.subType = RFXComUndecodedRFMessage.SubType.ARC;
-        msg.seqNbr = 1;
-        msg.rawPayload = HexUtils.hexToBytes("000102030405060708090A0B0C0D0E0F101112131415161718191A1B1C1D1E1F2021");
+        assertThrows(RFXComMessageTooLongException.class,
+                () -> testMessage("25030101000102030405060708090A0B0C0D0E0F101112131415161718191A1B1C1D1E1F2021",
+                        RFXComUndecodedRFMessage.SubType.ARC, 0x01,
+                        "000102030405060708090A0B0C0D0E0F101112131415161718191A1B1C1D1E1F2021"));
 
-        assertThrows(RFXComMessageTooLongException.class, () -> msg.decodeMessage());
+        // RFXComGenericDeviceConfiguration config = new RFXComGenericDeviceConfiguration();
```

Comment:
Should this be removed or commented out?

Reply:
Removed, thanks :)